### PR TITLE
Development Guide: use script/godep-restore.sh

### PR DESCRIPTION
**Pull request by kadel**

### CODE REVIEW

**Comment by surajssd** [comment]
Also one question @kadel 
In root directory if I do `godep save` it fails and when I removed the `vendor` and `Godeps` it didn't create new `vendor` and `Godeps`, I had to fo `godep save ./cli/main` and it ran alright.

**Comment by kadel** [comment]
you have to run `godep save ./...` from project root directory

**Comment by surajssd** [comment]
Actually I got confused because docs everywhere suggests just `godep save`.

**Comment by kadel** [comment]
@surajssd check this: https://github.com/tools/godep#multiple-packages

**Comment by surajssd** [comment]
@kadel thanks for the pointer

File: docs/development.md [hunk 1]
```diff
@@ -114,9 +114,12 @@ If it is not a valid version try, make sure you have updated the godep repo with
 
 ```console
 cd $GOPATH/src/github.com/skippbox
-godep restore
+script/godep-restore.sh
 ```
 
+Reason for calling `script/godep-restore.sh` instead of just `godep restore` is that Kompose is using `github.com/openshift/kubernetes` instead of `github.com/kubernetes/kubernetes`.
```
Comment (ngtuna) [inline docs/development.md]: @kadel Is there any way to just using `godep restore` ? currently we are using openshift/kubernetes to support OpenShift provider, what if we upgrade kubernetes to next releases 1.4, 1.5, etc ?

Comment (kadel) [inline docs/development.md]: OpenShift team usually rebase `openshift/kubernetes` as soon as there is a new release of kubernetes.
But I believe that in future we will be able to switch back to `kubernetes/kubernetes` as changes that openshift depends on are going to be merged upstream.

**Pull request merged**